app/page-partials/my-dashboard/IntegrationInfoTabs.tsx

```diff
@@ -125,70 +140,73 @@ function IntegrationInfoTabs({ integration, state, dispatch }: Props) {
-                    )}
-                  </Grid.Col>
-                </Grid.Row>
-              </Grid>
-            </TabWrapper>
-          </Tab>
-          {isGold && (
-            <Tab eventKey="client-roles" title="Role Management">
-              <TabWrapper>
-                <br />
-                <div>
-                  Please visit our{' '}
-                  <Link external href="https://github.com/bcgov/sso-keycloak/wiki/Creating-a-Role">
-                    wiki page
-                  </Link>{' '}
-                  for more information on roles.
-                </div>
-                <ClientRoles selectedRequest={integration} />
+        <BottomPanelHeader>INTEGRATION DETAILS - {padStart(String(integration.id), 8, '0')}</BottomPanelHeader>
+        <Container>
+          <RequestTabs
+            activeKey={panelTab}
+            mountOnEnter={true}
+            unmountOnExit={true}
+            onSelect={(k: any) => dispatch($setPanelTab(k))}
+          >
+            <Tab eventKey="installation-json" title="Technical Details">
+              <TabWrapper short={false}>
+                <Grid cols={15}>
+                  <Grid.Row gutter={[]}>
+                    <Grid.Col span={7}>
+                      <InstallationPanel selectedRequest={integration} />
+                    </Grid.Col>
+                    <Grid.Col span={1}></Grid.Col>
+                    <Grid.Col span={7}>
+                      {awaitingBceidProd && (
+                        <>
+                          <br />
+                          <SubTitle>Access to Prod (TBD)</SubTitle>
+                          <br />
+                          <br />
+                          <BceidStatus request={integration} />
+                        </>
+                      )}
+                    </Grid.Col>
+                  </Grid.Row>
+                </Grid>
               </TabWrapper>
             </Tab>
-          )}
-          {isGold && (
-            <Tab eventKey="user-roles" title="Assign Users to Roles">
-              <TabWrapper>
-                <UserRoles selectedRequest={integration} />
-              </TabWrapper>
-            </Tab>
-          )}
-          {!integration.publicAccess && (
-            <Tab eventKey="configuration-url" title="Secrets">
+            {isGold && (
+              <Tab eventKey="client-roles" title="Role Management">
+                <TabWrapper>
+                  <br />
+                  <div>
+                    Please visit our{' '}
+                    <Link external href="https://github.com/bcgov/sso-keycloak/wiki/Creating-a-Role">
+                      wiki page
+                    </Link>{' '}
+                    for more information on roles.
+                  </div>
+                  <ClientRoles selectedRequest={integration} />
+                </TabWrapper>
+              </Tab>
+            )}
+            {isGold && (
+              <Tab eventKey="user-roles" title="Assign Users to Roles">
```

Comment:
I believe we can group `client-roles` and `user-roles` tabs together under `isGold` condition.

Reply:
I did not change those 4 tabs.
I tried to put them together within the same quote under `isGold` condition, but the display has problem: either miss one tab, or both not showing.
Is it my structure wrong?

Reply:
Sorry. Missed your comment back. You will need ‘<></>’ as the wrapper in have more than one jsx elements in it.